src/components/ReduxFormSelect/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, FormControl } from '@edx/paragon';
+
+const ReduxFormSelect = ({
+  input, label: formLabel, disabled, options, description, meta: { touched, error },
+}) => {
+  const hasError = !!(touched && error);
```

Comment:
! equaling negation and !! ensuring boolean broke my brain a bit here, but I realize now this logic is totally fine and TIL "!!"

Reply:
Oh, yeah `!!` is great; without it this resolves to the error string which was making things weird. ;)